pool/connection_pool_test.go

```diff
@@ -329,11 +369,10 @@ func TestContextCancelInProgress(t *testing.T) {
 	}
 
 	connPool, err := pool.Connect(ctx, instances)
-	require.NotNilf(t, err, "expected err after ctx cancel")
-	assert.Truef(t, strings.Contains(err.Error(), "operation was canceled"),
-		fmt.Sprintf("unexpected error, expected to contain %s, got %v",
-			"operation was canceled", err))
-	require.Nilf(t, connPool, "conn is not nil after ctx cancel")
+	assert.NoError(t, err, "expected err after ctx cancel")
+	assert.NotNil(t, connPool)
+
+	connPool.Close()
```

Comment:
```suggestion
	defer connPool.Close()
```
This should prevent `svacer` warning about handle leak. Of course, if `svacer` will checks the tests. Minor, up to you.

Reply:
Thank you, updated.